### PR TITLE
FEATURE: Use diffhtml to update composer preview

### PR DESCRIPTION
**Pull request by nbianca**
Displaying videos, animated GIFs or any kind of rich content in preview
used to refresh on every keystroke, which could cause performance
problems.

### CODE REVIEW

**Comment by SamSaffron** [comment]
Can we put this behind an experimental site setting? I love the change, but want to limit the amount of risk here.

File: app/assets/javascripts/vendor.js [hunk 1]
```diff
@@ -9,6 +9,7 @@
 //= require popper.js
 //= require bootstrap-modal.js
 //= require caret_position
+//= require diffhtml.min.js
```
Comment (ZogStriP) [inline app/assets/javascripts/vendor.js:12]: I wonder if we could include it based on the value of the site setting?

File: app/assets/javascripts/discourse/app/lib/link-hashtags.js [hunk 2]
```diff
@@ -15,21 +14,19 @@ export function linkSeenHashtags($elem) {
...
-      }
-
-      const lowerSlug = slug.toLowerCase();
-      if (categoryHashtags[lowerSlug] && !hasTagSuffix) {
-        replaceSpan($(hashtag), slug, categoryHashtags[lowerSlug]);
-      } else if (tagHashtags[lowerSlug]) {
-        replaceSpan($(hashtag), slug, tagHashtags[lowerSlug]);
-      }
-    });
+  $hashtags.each((index, hashtag) => {
```
Comment (ZogStriP) [inline app/assets/javascripts/discourse/app/lib/link-hashtags.js:17]: Why was this removed from the `afterRender` callback? I guess it should depend on the site setting?

Comment (nbianca) [reply app/assets/javascripts/discourse/app/lib/link-hashtags.js:17]: I do not think this is necessary because it is called from inside a `afterRender` block already. 

```
d-editor.js / _updatePreview()
-> calls schedule("afterRender", ...)
-> calls composer-editor.js / previewUpdated
-> calls linkSeenHashtags
```

**Review by ZogStriP — APPROVED** [review]
LGTM
